### PR TITLE
Transport/opt

### PR DESCRIPTION
**Pull request by SylvainPlessis**
supersedes PR #93.

graph

Changes are made as explicit as possible with commit messages.  Only ```WilkeTranportEvaluator``` and ```PhysicalSet``` takes ```KineticsConditions```.

### CODE REVIEW

File: src/core/include/antioch/kinetics_conditions.h [hunk 1]
```diff
@@ -109,7 +115,14 @@ namespace Antioch{
...
   {
-     return _temperature;
+     return _temperature.T;
+  }
+
+  template <typename StateType, typename VectorStateType>
+  inline
+  const StateType & KineticsConditions<StateType,VectorStateType>::Tvib() const
+  {
+     return _temperature.T;
```
Comment (pbauman) [inline src/core/include/antioch/kinetics_conditions.h]: Maybe I'm missing something, but it seems like this method is for vibrational temperature and we're returning translational temperature.

Comment (SylvainPlessis) [inline src/core/include/antioch/kinetics_conditions.h]: Yep, and so far, all we have is Tv = T.

**Comment by SylvainPlessis** [comment]
Just ping me when we want this one merged, the conflicts were expected and should be on the method's name (``` StateType & T``` v/s ```KineticsConditions<> & conditions```).

I'll merge master in it at the last moment, to be sure it requires only one merge.

**Comment by roystgnr** [comment]
Putting it through the tests at the moment.

File: src/core/include/antioch/kinetics_conditions.h (hunk 1 above)
Comment (pbauman) [inline src/core/include/antioch/kinetics_conditions.h]: Let's at least put a comment in to that effect so we don't inadvertently deceive anyone.

**Comment by pbauman** [comment]
Barring adding the comment, I'm good with this if Roy's tests pass.

**Comment by roystgnr** [comment]
All good; ready to merge.

**Comment by SylvainPlessis** [comment]
On it.

**Pull request merged**